src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -73,8 +76,8 @@ const ApplyForBenefits = ({
     [shouldGetDD4EDUStatus, getDD4EDUStatus],
   );
 
-  const showHealthCare = !isPatient && !isInESR;
-  const showEducation = !hasDD4EDU;
+  const hideHealthCareBenefitInfo = hasHCAInProgress || isPatient || isInESR;
```

Comment:
I flipped these boolean names and how they work. By default we want to show all the info to all users. Writing things that way makes it a little clearer when we want to make an exception to the rule and hide a particular bit of benefit info.